frontend/app/page.tsx

```diff
@@ -254,16 +267,35 @@ export default function SignUp() {
                                 aria-label="password requirements"
                                 component="div"
                                 variant="inherit"
-                                className="px-4"
+                                className="p-4"
                             >
-                                <p className="-mb-2">Password must contain:</p>
-                                <ul>
-                                    <li>At least 10 characters</li>
-                                    <li>At least 1 lowercase character</li>
-                                    <li>At least 1 uppercase character</li>
-                                    <li>At least 1 number</li>
-                                    <li>At least 1 special character</li>
-                                </ul>
+                                <PasswordChecklist
```

Comment:
**praise**: Good use of react components! It makes the code so clean and easy to read.